### PR TITLE
feat: add partners page

### PR DESCRIPTION
**Pull request by maban**
This PR adds the partners page to the pages repo. In order to publish it properly, it will need to be hooked up on Akamai.

Merging this PR will not make any visible changes until _this_ PR is merged: https://github.com/Snyk/registry/pull/1480

So you may want to merge prematurely for demo purposes. Only people with the link will be able to see it.

All of the text is dummy content, please change it in this file: https://github.com/Snyk/pages/pull/20/files#diff-5e381db996e2fc9d7db9c45b04d472fc and ensure you maintain proper formatting (specifically, no quotes within quotes).

screen shot 2016-12-14 at 18 19 58

## Contact link

screen shot 2016-12-19 at 17 02 14

### CODE REVIEW

**Comment by geva** [comment]
Partners are rendering vertically on Safari (looks ok on Chrome, Firefox though)

image

**Comment by geva** [comment]
Could you also add a 'Partners' link to the menu bar at the top, probably to the right of VulnDB?

File: partners.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+partners:
+  - partner: "BitHound"
+    url: "https://www.bithound.io/"
+    logo: "https://res.cloudinary.com/snyk/image/upload/v1481724345/partners/bithound.png"
+    text: "Beyond security, dependencies also introduce quality risks, such as outdated, unused and deprecated dependencies. Use <a href='https://www.bithound.io/'>bitHound</a> to easily manage your dependencies and code quality."
+  - partner: "Nodesource"
+    url: "https://nodesource.com/"
+    logo: "https://res.cloudinary.com/snyk/image/upload/partners/nodesource.png"
+    text: "<a href='https://nodesource.com/'>NodeSource</a> is The Enterprise Node.js Company that provide Node.js support, training, and consulting, trusted by the world's biggest brands."
```
Comment (geva) [inline partners.html]: provide --> provides

**Comment by maban** [comment]
@geva This is already implemented in this PR, which can be pushed when the page is live https://github.com/Snyk/registry/pull/1480

I'll look into the Safari issue tomorrow.

**Comment by maban** [comment]
@geva @AH7 Please feel free to dive into the content – I just put dummy text in and I don't think "Howdy, partner" is a very good title!

**Comment by geva** [comment]
image

Navigation bar doesn't look good (i.e doesn't convert into the "Menu") on firefox,chrome

File: partners.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    logo: "https://res.cloudinary.com/snyk/image/upload/partners/risingstack.png"
+    text: "<a href='https://risingstack.com/'>RisingStack</a> is an enterprise-grade Node.js consulting and development. Micro-services-based API and Full Stack JavaScript development with AngularJS and React."
+  - partner: "VersionEye"
+    url: "https://www.versioneye.com/"
+    logo: "https://res.cloudinary.com/snyk/image/upload/v1481738332/partners/versioneye.png"
+    text: "<a href='https://www.versioneye.com/'>VersionEye</a> notifies you about out-dated dependencies, security vulnerabilities and license violations in your Git repositories"
+
+---
+
+We work with some really great teams to help make your code more secure. You can read about more of them here.
```
Comment (geva) [inline partners.html]: "You can read about more of them here" ---> "You can read about them here"

**Comment by maban** [comment]
@geva ignore that, the pages repo is a demo and doesn't pull in all of our code so it's missing some of the JavaScript. This will work on the live site.

File: partners.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    logo: "https://res.cloudinary.com/snyk/image/upload/partners/jfrog.png"
+    text: "<a href='https://www.jfrog.com/'>JFrog</a> provides world-class infrastructure for software management and distribution"
+  - partner: "RisingStack"
+    url: "https://risingstack.com/"
+    logo: "https://res.cloudinary.com/snyk/image/upload/partners/risingstack.png"
+    text: "<a href='https://risingstack.com/'>RisingStack</a> is an enterprise-grade Node.js consulting and development. Micro-services-based API and Full Stack JavaScript development with AngularJS and React."
+  - partner: "VersionEye"
+    url: "https://www.versioneye.com/"
+    logo: "https://res.cloudinary.com/snyk/image/upload/v1481738332/partners/versioneye.png"
+    text: "<a href='https://www.versioneye.com/'>VersionEye</a> notifies you about out-dated dependencies, security vulnerabilities and license violations in your Git repositories"
```
Comment (geva) [inline partners.html]: Missing '.' at the end

File: partners.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    logo: "https://res.cloudinary.com/snyk/image/upload/partners/nodesource.png"
+    text: "<a href='https://nodesource.com/'>NodeSource</a> is The Enterprise Node.js Company that provide Node.js support, training, and consulting, trusted by the world's biggest brands."
+  - partner: "Twistlock"
+    url: "https://www.twistlock.com/"
+    logo: "https://res.cloudinary.com/snyk/image/upload/partners/twistlock.png"
+    text: "<a href='https://www.twistlock.com/'>Twistlock’s</a> technologies are built for containers — from hardening images in development to protecting containers in production — everything we do fits seamlessly in the container environment."
+  - partner: "JFrog"
+    url: "https://www.jfrog.com/"
+    logo: "https://res.cloudinary.com/snyk/image/upload/partners/jfrog.png"
+    text: "<a href='https://www.jfrog.com/'>JFrog</a> provides world-class infrastructure for software management and distribution"
```
Comment (geva) [inline partners.html]: Missing '.' at the end

File: partners.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  - partner: "RisingStack"
+    url: "https://risingstack.com/"
+    logo: "https://res.cloudinary.com/snyk/image/upload/partners/risingstack.png"
+    text: "<a href='https://risingstack.com/'>RisingStack</a> is an enterprise-grade Node.js consulting and development. Micro-services-based API and Full Stack JavaScript development with AngularJS and React."
```
Comment (geva) [inline partners.html]: "consulting and development" --> "consulting and development firm"

File: partners.html (hunk 1 above)
Comment (geva) [inline partners.html]: Quick google search shows "largest brands" is used more than "biggest brands"

File: partners.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    logo: "https://res.cloudinary.com/snyk/image/upload/v1481724345/partners/bithound.png"
+    text: "Beyond security, dependencies also introduce quality risks, such as outdated, unused and deprecated dependencies. Use <a href='https://www.bithound.io/'>bitHound</a> to easily manage your dependencies and code quality."
+  - partner: "Nodesource"
+    url: "https://nodesource.com/"
+    logo: "https://res.cloudinary.com/snyk/image/upload/partners/nodesource.png"
+    text: "<a href='https://nodesource.com/'>NodeSource</a> is The Enterprise Node.js Company that provide Node.js support, training, and consulting, trusted by the world's biggest brands."
+  - partner: "Twistlock"
+    url: "https://www.twistlock.com/"
+    logo: "https://res.cloudinary.com/snyk/image/upload/partners/twistlock.png"
+    text: "<a href='https://www.twistlock.com/'>Twistlock’s</a> technologies are built for containers — from hardening images in development to protecting containers in production — everything we do fits seamlessly in the container environment."
```
Comment (geva) [inline partners.html]: This is the place on the page that has 1st tense "everything *we* do..."
Suggest to replace "everything we do" --> "everything Twistlock does"

**Pull request merged**